### ISSUE TITLE
Elevated circulating erythropoietin

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6718]
**Preferred term label:**
Elevated circulating erythropoietin

**Synonyms**

**Definition (free text, please give PubMed ID)**
Increased amount of erythropoietin in the blood circulation. Erythropoietin is a glycoprotein hormone produced by the peritubular cells of the kidney that stimulates red blood cell production. 
PMID:30725682

**Parent term (use hpo.jax.org/app)**
Abnormal circulating hormone level HP:0003117

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 OMIM:611783 
PMID:18184961  1/1
PMID:18378852  5/5

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
